dev/refresh_source.py

```diff
@@ -434,7 +434,8 @@ def write_deck_run_script(self, repository):
 cd "$d"
 LOG_DIR=${{LOG_DIR:-../logs}}
 
-bash -c "(./start.sh >> '$LOG_DIR/{name}.log') 2>&1\
+npm install >& "$LOG_DIR/deck.log"
```

Comment:
I patched in your changes to my already-generated start-dev.sh file. It looks like so:
```
#!/bin/bash
d=$(dirname "$0")
cd "$d"
LOG_DIR=${LOG_DIR:-../logs}

npm install >& "$LOG_DIR/deck.log"
bash -c "(npm run start-dev-server >> '$LOG_DIR/{name}.log') 2>&1\
 | tee -a '$LOG_DIR/{name}.log' >& '$LOG_DIR/{name}.err' &"
```